Sources/Apollo/GraphQLMap.swift

```diff
@@ -0,0 +1,19 @@
+public typealias GraphQLMap = [String: JSONEncodable?]
+
+fileprivate extension Dictionary where Key == String, Value == JSONEncodable? {
+  var withNilValuesRemoved: Dictionary<String, JSONEncodable> {
+    filter { $0.value != nil }
```

Comment:
Any particular reason to use this instead of `compactMap { $0.value }`?

Reply:
Nope, I had that and changed it. Same thing, just what we are doing here is filtering out nils, so I figured this was more expressive of the intent.

Reply:
I'd argue `compactMap` also expresses that intent, but I'm not dying on this hill ☠️

Reply:
Yeah, I really truly don't care either way myself either.